.github/workflows/python-app.yml

```diff
@@ -99,6 +109,7 @@ jobs:
         pip install pylint
     - name: Lint with pylint
       run: |
+        . testenv/bin/activate
```

Comment:
Doesn't this need  a `python -m venv testenv` also?

Reply:
The testenv venv is created in the previous step: "Install dependencies"